src/api/roles/client/client.py

```
@@ -610,25 +677,33 @@ def get_my_coach(db = Depends(get_session), acc: Account = Depends(get_client_ac
     if acc is None:
         raise HTTPException(404, detail="Account not found")
 
-    coach_request = db.query(ClientCoachRequest).filter(
-        ClientCoachRequest.client_id == acc.client_id,
-        ClientCoachRequest.is_accepted == True
-    ).order_by(ClientCoachRequest.last_updated).first()
+    coach_request = db.exec(
+        select(ClientCoachRequest)
+        .where(
+            ClientCoachRequest.client_id == acc.client_id,
+            ClientCoachRequest.is_accepted == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `ClientCoachRequest.is_accepted:` for truth checks

Replace with `ClientCoachRequest.is_accepted`

(E712)